### PR TITLE
[DS-1744] Changes to CODE_OF_CONDUCT and CONTRIBUTORS

### PR DESCRIPTION
**Pull request by kkanishkan**
https://waveaccounting.atlassian.net/browse/DS-1744

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# Code of Conduct
+
+This code of conduct sets the expectations to participate in the following open source community managed by Wave Apps. Anyone violating this code of conduct may be banned from the community.
```
Comment (eballerini) [inline CODE_OF_CONDUCT.md:3]: We may need to refer to Wave as Wave Financial Inc. instead of Wave Apps.

File: CODE_OF_CONDUCT.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+With this project, Wave Apps is hoping to create a community that is welcoming, friendly, and patient.
+ 
+## Our Responsibilities
+ 
+The maintainers are responsible for clarifying the standards of acceptable behaviour and are expected to corrective action in response to instances of abuse of the Code of Conduct.
```
Comment (Tarun-S) [inline CODE_OF_CONDUCT.md:8]:
```suggestion
The maintainers are responsible for clarifying the standards of acceptable behaviour and are expected to take corrective action in response to instances of abuse of the Code of Conduct.
```

File: CODE_OF_CONDUCT.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+ 
+Maintainers have the right to remove, edit, or reject comments, commits, code, issues, and other contributions that are not aligned with the Code of Conduct. Maintainers may ban any contributor temporarily or permanently for behaviours they deem inappropriate, threatening, offensive, or harmful.
+ 
+## Scope
+This Code of Conduct both within the project space and in the public where someone is representing the project or the community.
```
Comment (Tarun-S) [inline CODE_OF_CONDUCT.md:13]:
```suggestion
This Code of Conduct applies both within the project space and in the public where someone is representing the project or the community.
```

File: CONTRIBUTORS.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
+## Contributing to Avro to Python Types
+ 
+Thank you for your interest in contributing to this project! All kinds of contributions are welcome; commenting, reviewing and sending pull requests! We hope to build a vibrant community as we grow the project.
+ 
+Before contributing please visit the [Open Source Guides](https://opensource.guide/) to learn more about contributing to an open-source project.
```
Comment (Tarun-S) [inline CONTRIBUTORS.md:5]:
```suggestion
Before contributing, please visit the [Open Source Guides](https://opensource.guide/) to learn more about contributing to an open-source project.
```

File: CONTRIBUTORS.md [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+
+All kinds of contributions are welcome. When contributing code please use [open issues](https://github.com/waveaccounting/avro-to-python-types/issues); code changes, comments are welcome. QA, filing bug reports and creating issues are very welcome (if you can include as much detail as possible and screenshots, it would be appreciated!).
+
+## Code Changes
+
+Any code change must be submitted as a pull request. The description should refer to the issue that the change is addressing with a description of what the code does. The pull request should also contain tests.
+
+## Questions
+
+If you have any questions, create an [issue](https://github.com/waveaccounting/avro-to-python-types/issues). Using the 'Report Content' option in Github, content that violates the Code of Conduct can be reported.
```
Comment (Tarun-S) [inline CONTRIBUTORS.md:19]: The second line seems a little redundant, as it's already mentioned in the Enforcement section.

File: CODE_OF_CONDUCT.md (hunk 1 above)
Comment (Tarun-S) [inline CODE_OF_CONDUCT.md:3]:
```suggestion
This Code of Conduct sets the expectations to participate in the following open source community managed by Wave Apps. Anyone violating this code of conduct may be banned from the community.
```

Comment (dangreenisrael) [reply CODE_OF_CONDUCT.md:3]: I'll check with P&C

File: CODE_OF_CONDUCT.md [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+ 
+Maintainers have the right to remove, edit, or reject comments, commits, code, issues, and other contributions that are not aligned with the Code of Conduct. Maintainers may ban any contributor temporarily or permanently for behaviours they deem inappropriate, threatening, offensive, or harmful.
+ 
+## Scope
+This Code of Conduct applies both within the project space and in the public where someone is representing the project or the community.
+
+## Enforcement
+Instances of abusive, harassing, or other unacceptable behaviour may be reported by using the 'Report Content' option in Github. All complaints will be reviewed and investigated and will result in a response that is deemed necessary and appropriate to the circumstances. The project team is obligated to maintain confidentiality with regard to the reporter of an incident. Further details of specific enforcement policies may be posted separately.
+Project maintainers who do not follow or enforce the Code of Conduct in good faith may face temporary or permanent repercussions as determined by other members of the project's leadership.
```
Comment (dangreenisrael) [inline CODE_OF_CONDUCT.md:17]: I think we can drop this line as maintainers will be Wave employees, at least for the foreseeable future.

**Review by Tarun-S — APPROVED** [review]
LGTM!

**Review by dangreenisrael — DISMISSED** [review]
LGTM

**Review by dangreenisrael — APPROVED** [review]

**Pull request merged**